composer.json

```diff
@@ -17,7 +17,7 @@
         "keboola/php-csv-db-import": "~2.3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.6.*",
+        "phpunit/phpunit": "^7.0",
```

Comment:
Why the upgrade? And why not upgrade anything else (cs, phpstan)? I have to say I much prefer to create separate PR for any non-related changes.